docs/installation.rst

```diff
@@ -39,29 +39,7 @@ this method by following the steps described below.
     $ conda create --name tardis --file conda-{platform}-64.lock
     $ conda activate tardis
 
-3. a. Non-developers can install the latest release from ``conda-forge`` with the ``--no-deps`` flag,
-
-      .. warning::
-
-        Currently the conda forge installation doesn't work. It's recommended to install from the specific releases using pip-
-        
-        `$ pip install git+https://github.com/tardis-sn/tardis.git@{tag}`
-
-        For example- 
-        
-        `pip install git+https://github.com/tardis-sn/tardis.git@release-2023.09.17`
-    
-      ::
-
-        $ conda install tardis-sn --channel conda-forge --no-deps
-
-      or trying the most recent, unreleased changes from upstream.
-
-      ::
-
-        $ pip install git+https://github.com/tardis-sn/tardis.git@master
-
-   b. Instead, developers should `fork the repository <https://github.com/tardis-sn/tardis/fork>`_, configure
+3. a. Developers should `fork the repository <https://github.com/tardis-sn/tardis/fork>`_, configure
```

Comment:
There should be a somewhere that the conda.lock file downloaded in 1. from the latest (or really any) releasemight not match the conda.lock file from the current master branch/ the forked repository. See issue #2429

Developers should imo create their conda environment using the conda lockfile from the cloned/ forked tardis repository, not the release.

Reply:
Please change line 33 to "$ wget -q https://github.com/tardis-sn/tardis/blob/master/conda-{platform}-64.lock"